WordPress/Classes/ViewRelated/Post/Revisions/Browser/RevisionBrowserState.swift

```diff
@@ -0,0 +1,26 @@
+class RevisionBrowserState {
+    typealias RevisionSelectedBlock = (Revision) -> Void
+
+    let revisions: [Revision]
+    var currentIndex: Int
```

Comment:
Consider making this `private(set)`, since the class is designed to user `increaseIndex()` and `decreaseIndex()`.